src/main/java/org/springframework/data/redis/cache/RedisCacheConfiguration.java

```diff
@@ -108,14 +110,18 @@ public static RedisCacheConfiguration defaultCacheConfig(@Nullable ClassLoader c
 
 		registerDefaultConverters(conversionService);
 
-		return new RedisCacheConfiguration(TtlFunction.persistent(), DEFAULT_CACHE_NULL_VALUES, DEFAULT_USE_PREFIX,
+		return new RedisCacheConfiguration(TtlFunction.persistent(),
+				DEFAULT_CACHE_NULL_VALUES,
+				DEFAULT_ENABLE_TTI_EXPIRATION,
+				DEFAULT_USE_PREFIX,
 				CacheKeyPrefix.simple(),
 				SerializationPair.fromSerializer(RedisSerializer.string()),
 				SerializationPair.fromSerializer(RedisSerializer.java(classLoader)),
 				conversionService);
 	}
 
 	private final boolean cacheNullValues;
+	private final boolean enableTtiExpiration;
```

Comment:
Let's expand on the name to `enableTimeToIdle` instead of `Tti`.

Reply:
Done.

I also renamed `isTtiExpirationEnabled()` to `isTimeToIdleEnabled()` and `enableTtiExpiration()` to `enableTimeToIdle()`.